unstructured/ingest/connector/chroma.py

```diff
@@ -0,0 +1,162 @@
+import itertools
+import json
+import typing as t
+import uuid
+from dataclasses import dataclass
+
+from unstructured.ingest.error import DestinationConnectionError
+from unstructured.ingest.interfaces import (
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.staging.base import flatten_dict
+from unstructured.utils import requires_dependencies
+
+if t.TYPE_CHECKING:
+    from chromadb import Collection as ChromaCollection
+
+    # from chromadb.config import Settings as ChromaSettings
+
+
+@dataclass
+class SimpleChromaConfig(BaseConnectorConfig):
+    collection_name: str
```

Comment:
How does this authenticate when communicating with something not on the user's local? Should contain that within an `AccessConfig`.

Reply:
@rbiseck3 Was definitely looking into this. You think just `headers` and `settings` belong in AccessConfig or also `port, host, ssh`? I chose just `headers, settings` due to the `AccessConfig` docstring.

https://github.com/chroma-core/chroma/blob/main/chromadb/__init__.py